src/MountedTraversal.js

```diff
@@ -177,6 +178,12 @@ export function parentsOfInst(inst, root) {
   return pathToNode(inst, root).reverse();
 }
 
+export function instMatchesObjectProps(inst, props) {
+  if (!isDOMComponent(inst)) return false;
+  const node = getNode(inst);
+  return !isEmpty(props) && isMatch(propsOfNode(node), props);
```

Comment:
if they are actually passing in an empty object, then matching all nodes actually seems like the reasonable response to me?

I agree this doesn't seem like a normal use case, although I can imagine some code that builds up some object dynamically, and in some cases might build up to nothing...

I don't feel strongly about this, but I don't see the harm in having `{}` match all nodes. It seems like the expected behavior, despite being a "corner-case" type input?